### PR TITLE
Fixes #16435 - Replace underscore with lodash

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ehelms** [comment]
Looks good to me -- @waldenraines any concerns?

**Comment by waldenraines** [comment]
I don't see where the lodash library was ever actually added to source control.  Unfortunately because of build requirements we have to store our javascript libraries in source control.  Did you ever run a `grunt bower:update`?

**Comment by ehelms** [comment]
@waldenraines I believe the idea is that lodash is part of the core Foreman JS stack and we can thus rely on using it from there. Thus we only need it declared to ensure its available during unit testing.

**Comment by waldenraines** [comment]
> @waldenraines I believe the idea is that lodash is part of the core Foreman JS stack and we can thus rely on using it from there. Thus we only need it declared to ensure its available during unit testing.
> Add your review

Ah, okay, didn't realize foreman included it.

File: app/assets/javascripts/bastion/components/bst-edit.directive.js [hunk 1]
```diff
@@ -290,13 +290,13 @@ angular.module('Bastion.components')
 
         getIds = function (models) {
             models = models || [];
-            return _.pluck(models, "id");
+            return _.map(models, "id");
```
Comment (waldenraines) [inline app/assets/javascripts/bastion/components/bst-edit.directive.js]: Wouldn't these be better written as `_.property(...)`?

Comment (xprazak2) [inline app/assets/javascripts/bastion/components/bst-edit.directive.js]: I do not think it is the same, `_.property` returns a function, [jsbin](https://jsbin.com/difixo/2/edit?html,console) seems to agree...

**Comment by xprazak2** [comment]
Yes, katello will require [changes](https://github.com/Katello/katello/pull/6375) as well.

**Comment by jlsherrill** [comment]
@waldenraines ping on this PR

**Comment by waldenraines** [comment]
Thanks @xprazak2, sorry about the delay in merging.

**Comment by xprazak2** [comment]
Thanks @waldenraines, could you release a new Bastion version with this change pls? I believe it is needed to get the Katello part in, because Katello uses Bastion from rubygems on Jenkins...

**Comment by waldenraines** [comment]
> Thanks @waldenraines, could you release a new Bastion version with this change pls? I believe it is needed to get the Katello part in, because Katello uses Bastion from rubygems on Jenkins...

@ehelms, do you mind?

**Comment by ehelms** [comment]
Done.

**Pull request merged**